backend/fms_core/tests/test_template_importers/sample_pooling_planning_test.py

```diff
@@ -0,0 +1,190 @@
+                        self.assertEqual(csv_content[4][2], "TubeDst1")
+                        self.assertEqual(csv_content[4][3], "1")
+                        self.assertEqual(csv_content[4][4], "0")
+                        self.assertEqual(csv_content[4][5], "50.000")
+                        # Fifth sample
+                        self.assertEqual(csv_content[5][0], "Src1")
+                        self.assertEqual(csv_content[5][1], "10")
+                        self.assertEqual(csv_content[5][2], "TubeDst1")
+                        self.assertEqual(csv_content[5][3], "1")
+                        self.assertEqual(csv_content[5][4], "0")
+                        self.assertEqual(csv_content[5][5], "40.000")
+                        # Sixth sample
+                        self.assertEqual(csv_content[6][0], "Src1")
+                        self.assertEqual(csv_content[6][1], "4")
+                        self.assertEqual(csv_content[6][2], "TubeDst2")
+                        self.assertEqual(csv_content[6][3], "2")
+                        self.assertEqual(csv_content[6][4], "0")
+                        self.assertEqual(csv_content[6][5], "80.000")
+                        # Seventh sample
+                        self.assertEqual(csv_content[7][0], "Src1")
+                        self.assertEqual(csv_content[7][1], "5")
+                        self.assertEqual(csv_content[7][2], "TubeDst2")
+                        self.assertEqual(csv_content[7][3], "2")
+                        self.assertEqual(csv_content[7][4], "0")
+                        self.assertEqual(csv_content[7][5], "80.000")
+                        # Eighth sample
+                        self.assertEqual(csv_content[8][0], "Src1")
+                        self.assertEqual(csv_content[8][1], "17")
+                        self.assertEqual(csv_content[8][2], "TubeDst2")
+                        self.assertEqual(csv_content[8][3], "2")
+                        self.assertEqual(csv_content[8][4], "0")
+                        self.assertEqual(csv_content[8][5], "25.000")
+                        # ninth sample
+                        self.assertEqual(csv_content[9][0], "Src1")
+                        self.assertEqual(csv_content[9][1], "61")
+                        self.assertEqual(csv_content[9][2], "TubeDst2")
+                        self.assertEqual(csv_content[9][3], "2")
+                        self.assertEqual(csv_content[9][4], "0")
+                        self.assertEqual(csv_content[9][5], "40.000")
+                        # tenth sample
+                        self.assertEqual(csv_content[10][0], "Src1")
+                        self.assertEqual(csv_content[10][1], "93")
+                        self.assertEqual(csv_content[10][2], "TubeDst2")
+                        self.assertEqual(csv_content[10][3], "2")
+                        self.assertEqual(csv_content[10][4], "0")
+                        self.assertEqual(csv_content[10][5], "100.000")
```

Comment:
You might have benefited from the use of an array or dictionary so you can assert in a loop. Although, that might make finding a specific assertion error a bit challenging. Anyhow, there's gotta be a way.